### ISSUE TITLE
Zooming slowly pulls Southeast.

### ISSUE THREAD

**Issue by thehack** [craftyjs/Crafty#339]
If I do:  

``` javascript
Crafty.viewport.zoom(scale, center.x, center.y, numberUnder100);
```

It works as expected, but if I do:

``` javascript
Crafty.viewport.zoom(scale, center.x, center.y,numberOver100);
```

It moves the camera to the Southeast of the map. See http://jsfiddle.net/czeeA/14/

The end result is you can't do a slow zoom over a couple seconds to create a cutscene from still images.

**Comment by mpetrovic** [comment]
I'll take a look at this tonight.

**Comment by starwed** [comment]
To be clear, if you use DOM entities it doesn't zoom at all!  Just the panning.

Amusingly, if you use Canvas it does zoom, but also pulls to the NW of the target point.  ([fiddle](http://jsfiddle.net/czeeA/22/))

**Comment by starwed** [comment]
Update: I'm working on a rewrite of the viewport animations that fixes this.

**Comment by starwed** [comment]
Should now be fixed by #648!
